docs/data/sdks/marketing-analytics-browser/migration.md

```diff
@@ -0,0 +1,103 @@
+
+### Terminology
+
+* `@amplitude/marketing-analytics-browser@1`: Marketing Analytics Browser SDK
+* `@amplitude/analytics-browser@2`:  Browser SDK 2.0
+
+### Dependency
+
+dependency.md
+
+```diff
+{
+  "dependencies": {
+-     "@amplitude/marketing-analytics-browser": "^1"
++     "@amplitude/analytics-browser": "^2"
+  }
+}
+```
+
+### Default events tracking
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/default-event-tracking.md"
+
+### Marketing attribution tracking
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/marketing-attribution-tracking-1.md"
+
+Marketing Analytics Browser SDK by default, allows other subdomains to be tracked as referrer. If this is behavior is desired, refer to the code below.
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/marketing-attribution-tracking-2.md"
+
+### Page View tracking
+
+#### Moved `options.pageViewTracking` to `options.defaultTracking.pageViews`
+
+```diff
+  amplitude.init(API_KEY, undefined, {
+-   pageViewTracking: {
+-     trackOn: 'attribution',
+-     trackHistoryChanges: 'pathOnly', 
+-     eventType: 'Page View',
++   defaultTracking: {
++     pageViews: {
++       trackOn: 'attribution',
++       trackHistoryChanges: 'pathOnly',
++       eventType:'Page View',
++   }
+    },
+  });
+```
+
+#### Disabling page view tracking
+
+This provides a simpler and consistent interface to opt out of page view tracking.
+
+```diff
+  amplitude.init(API_KEY, undefined, {
+-   pageViewTracking: false,
++   defaultTracking: {
++     pageViews: false,
+    },
+  });
+```
+
+#### Updates on page view tracking
+
+* The event type of page view has been changed from `Page View` to `[Amplitude] Page Viewed`.
+* The event properties name also be updated. Check [here](https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events.) for more info.
+
+| property | <div class="big-column"> [Browser SDK 2.0](../) </div> | <div class="big-column"> [Marketing Analytics Browser SDK](../../typescript-browser/) <div> |
+| --- | --- | --- |
+| `Event Type` | `[Amplitude] Page Viewed` | `Page View` |
+| `Event Properties` | `page_domain` |  `[Amplitude] Page Domain` |
+| | `page_location` | `[Amplitude] Page Location` |
+| | `page_path` | `[Amplitude] Page Path` |
+| | `page_title` | `[Amplitude] Page Title` |
+| | `page_url` | `[Amplitude] Page URL` |
+
+### Cookie options
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/cookie-options.md"
+
+### Deprecates user agent client-side parsing
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/user-agent-parsing.md"
+
+### No to enums
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/no-enum.md" 
+
+### Simplified plugin interface
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/simplified-plugin-interface.md" 
```

Comment:
There are not included in [preview](https://pr-872.d19s7xzcva2mw3.amplifyapp.com/data/sdks/marketing-analytics-browser/migration/#no-to-enums) but I couldn't find anything wrong with it.
<img width="631" alt="image" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/31029607/c7cfc6dd-a161-4160-8c47-3104907c9b09">

Reply:
<img width="765" alt="Screen Shot 2023-08-07 at 11 56 40 AM" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/13028329/fcde13a9-8540-4c80-99a7-37ccfcf20a1c">
Thanks for pointing it out. Should be fixed in the new commit.

Reply:
My local env looks fine. Don't know what's going on in the developer center. 
 @justin-fiedler  Any idea on this?
<img width="1340" alt="Screen Shot 2023-08-07 at 12 11 19 PM" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/13028329/ac4e26ad-9611-443e-bc31-b9429604ccf2">

Reply:
The include show for me - https://pr-872.d19s7xzcva2mw3.amplifyapp.com/data/sdks/marketing-analytics-browser/migration/#no-to-enums